component/conprof/http/api.go

```diff
@@ -356,7 +356,7 @@ func queryAndDownload(c *gin.Context) error {
 }
 
 const downloadReadme = `
-To review the CPU profiling or heap profiling result interactively:
+To review the profile data which file name suffix is `.proto` interactively:
```

Comment:
Not sure whether the nested back-quote works.

Reply:
And it is better to use `that` or `whose` to replace `which`, we can ask help from i18n.